System.Web/NOTICES.txt

```diff
@@ -0,0 +1,16 @@
+re-motion Framework
+www.re-motion.org
+Copyright (c) rubicon IT GmbH, www.rubicon.eu
+
+This software embeds code from "ASP.NET WebForms".
+Copyright (c) Microsoft Corporation
+Original software available from "https://github.com/microsoft/referencesource" or "https://dotnet.microsoft.com/en-us/download/dotnet-framework".
+Relevant code code parts:
+      System.Web
+      System.Web.ApplicationServices
+      System.Web.DynamicData
+      System.Web.Entity
+      System.Web.Extensions
+      System.Web.Extensions.Data.Linq
+      System.Web.RegularExpressions
+      System.Web.Services
```

Comment:
```suggestion
* System.Web
* System.Web.ApplicationServices
* System.Web.DynamicData
* System.Web.Entity
* System.Web.Extensions
* System.Web.Extensions.Data.Linq
* System.Web.RegularExpressions
* System.Web.Services
```

Whitespace is usally stripped in web display. Using * as bullet point indicator, we also have an extra hint in case linewraps are lost.